### PR TITLE
SelemiumBasicsPart1 is completed

### PR DESCRIPTION
**Pull request by NadzeyaKiz**
SelemiumBasicsPart1 task is done

### CODE REVIEW

File: Code/SeleniumBasics/UnitTest1.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using static System.Net.WebRequestMethods;
+
+namespace SeleniumBasics
+{
+    public class SelemiumBasicsEpamPageTests
```
Comment (dnnyhlwdplc) [inline Code/SeleniumBasics/UnitTest1.cs]: please do not forget to rename class in solution explorer as well :)

File: Code/SeleniumBasics/UnitTest1.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            _driver = new ChromeDriver();
+            _driver.Manage().Window.Maximize();
+        }
+
+        [Test]
+        public void NavigateToEpamePageTest()
+        {            
+            _driver.Navigate().GoToUrl(_epamUrl);            
+            var actualUrl = _driver.Url;
+            Assert.AreEqual(_epamUrl, actualUrl);
```
Comment (dnnyhlwdplc) [inline Code/SeleniumBasics/UnitTest1.cs]: Add Assert messages :)

File: Code/SeleniumBasics/UnitTest1.cs [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        {
+            var howWeDoItUrl = "https://www.epam.com/how-we-do-it";
+            var ourWorkUrl = "https://www.epam.com/our-work";
+            _driver.Navigate().GoToUrl(howWeDoItUrl);
+            _driver.Navigate ().GoToUrl(ourWorkUrl);
+            _driver.Navigate().Refresh();
+            _driver.Navigate().Back();
+            var actualUrl = _driver.Url;
+
+            Assert.AreEqual(howWeDoItUrl, actualUrl);
```
Comment (dnnyhlwdplc) [inline Code/SeleniumBasics/UnitTest1.cs]: same :)

**Review by dnnyhlwdplc — APPROVED** [review]

File: Code/SeleniumBasics/UnitTest1.cs (hunk 1 above)
Comment (NadzeyaKiz) [reply Code/SeleniumBasics/UnitTest1.cs]: done

File: Code/SeleniumBasics/UnitTest1.cs (hunk 2 above)
Comment (NadzeyaKiz) [reply Code/SeleniumBasics/UnitTest1.cs]: done

File: Code/SeleniumBasics/UnitTest1.cs (hunk 3 above)
Comment (NadzeyaKiz) [reply Code/SeleniumBasics/UnitTest1.cs]: done

**Comment by NadzeyaKiz** [comment]
Selenium Basics1 task completed

**Pull request merged**